test/primitives-test.es6

```diff
@@ -2,6 +2,48 @@ import {expect} from './spec-helper';
 import minim from '../lib/minim';
 
 describe('Minim Primitives', function() {
+  describe('ElementType', () => {
+    context('when initializing', () => {
+      let el;
+
+      before(() => {
+        el = new minim.ElementType('element', null, {
+          id: 'foobar',
+          class: ['a', 'b'],
+          name: 'name',
+          title: 'Title',
+          description: 'Description'
+        });
+      });
+
+      it('should initialize the correct meta data', () => {
+        expect(el.meta.id.get()).to.equal('foobar');
+        expect(el.meta.class.toValue()).to.deep.equal(['a', 'b']);
```

Comment:
I wonder how we expect consumers of this API to know when to use `get` vs. `toValue`. Does it make sense to keep both?